test/type/array.swift

```diff
@@ -114,3 +114,10 @@ func passAssocType<T : HasAssocType>(_ t: T) {
   takesAssocType(t, [T.A](), [T.A?]())
 }
 
+// SR-11134
+
+let sr_11134_1 = [[1, 2, 3][0]] // ok
+let sr_11134_2 = [[1, 2, 3] [1]] // expected-warning {{unexpected subscript in array literal; did you mean to write two separate elements instead?}}
```

Comment:
Can you add a multi-line test case as well? The code you wrote should already handle that.

Reply:
Do you mean something like 

```
[
  [1, 2, 3] [1]
]
```

?

Reply:
It works for that, however you can't put the index on a new line otherwise the parser will complain about a missing separator, example:

```
[[1, 2, 3]<space>
[1]]
```